src/arch/ia32/descriptor/segment.rs

```diff
@@ -124,6 +124,25 @@ pub struct SegmentDescriptor {
 }
 
 impl SegmentDescriptor {
+
+    /// Create a new null [`SegmentDescriptor`].
+    pub const fn const_default() -> Self {
```

Comment:
Why wouldn't we call it simply `default` since we are not implementing the `Default` trait ? Would it clash with some types ?

Reply:
That was a personal choice to be consistent in all structures. I guess if we implement the `ConstDefault` trait we solve both problems.

Reply:
As discussed, we leave it like that